pkgs/top-level/python-packages.nix

```diff
@@ -6362,6 +6362,8 @@ self: super: with self; {
     inherit (pkgs) fuse;
   };
 
+  llm = callPackage ../development/python-modules/llm { };
```

Comment:
@SuperSandro2000  it is used mainly as a `cli`, [but it can be used as a library as well](https://llm.datasette.io/en/stable/python-api.html) (I plan to use it that way soon).